tests/unit/CoreVideoTest.php

```diff
@@ -70,6 +77,25 @@ public function test_retrieve_core_video_attributes() {
 				}
 			}
 		';
+	}
+
+	public function test_retrieve_core_video_attributes() {
+		$block_content = '
+			<!-- wp:video {"id":1636,"align":"wide","lock":{"move":true,"remove":true},"className":"test-css-class"} -->
+			<figure id="test-anchor" class="wp-block-video" style="margin-top:var(--wp--preset--spacing--50);margin-right:var(--wp--preset--spacing--50);margin-bottom:var(--wp--preset--spacing--50);margin-left:var(--wp--preset--spacing--50)">
+			<video autoplay loop muted poster="http://mysite.local/wp-content/uploads/2023/05/pexels-egor-komarov-14420089-scaled.jpg" preload="auto" src="http://mysite.local/wp-content/uploads/2023/07/pexels_videos_1860684-1440p.mp4" playsinline></video>
+			<figcaption class="wp-element-caption">Sample caption</figcaption></figure>
+			<!-- /wp:video -->';
+
+		// Update the post content with the block content.
+		wp_update_post(
+			[
+				'ID'           => $this->post_id,
+				'post_content' => $block_content,
+			]
+		);
+
+		$query  = $this->query();
 		$actual = graphql( [ 'query' => $query ] );
```

Comment:
Use the same syntax/spacing/_actual_ asserts from our other approved tests.

Reply:
Oh-no! my bad, I assumed to build on top of their existing code structure 🙇 ( which was not having the basic asserts/ comments ) . I have fixed these in my latest work ✅

Reply:
Yeah it's a tricky line. Rationale here is that the existing test internals are not up to par, so we ask ourselves how would they write this if they had implemented it 'correctly'. We're keeping their naming conventions (e.g. method names) and structure (separating frag from query) while _fixing_ the asserts, testing an explicit post instead of relying on test state, etc.